### PR TITLE
fix: add secure mysql db samples, fixes RHOAIENG-5436

### PR DESCRIPTION
**Pull request by dhirajsb**
## Description

Added secure mysql db samples without and with istio tls
Fixes RHOAIENG-5436
Fixes RHOAIENG-6713

## How Has This Been Tested?

Tested samples manually, by first creating the DB cert using `make certificates` and then the appropriate `kubectl apply -k` command. 

## Merge criteria:

- [X] The commits and have meaningful messages; the author will squash them [after approval](https://github.com/opendatahub-io/opendatahub-community/blob/main/contributor-cheatsheet.md#:~:text=Usually%20this%20is%20done%20in%20last%20phase%20of%20a%20PR%20revision) or will ask to merge with squash.
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by dhirajsb** [comment]
@nehachopra27 @tonyxrmdavidson we should use these secure mysql samples in QE tests as that's closer to what users will deploy in production.

**Review by tarilabs — DISMISSED** [review]
lgtm, I notice there is some little mods to pgsql but I believe those are fine too thanks!

**Review by isinyaaa — DISMISSED** [review]

**Comment by rareddy** [comment]
1. Is there a procedure to update or rotate the certificates when they expire?
2. Is there a procedure for a user to bring in their custom certificates instead of self-generated?

Both could be just documentation details but be good to note for the users.

File: config/samples/postgres/postgres-db.yaml [hunk 1]
```diff
@@ -72,7 +72,9 @@ items:
           livenessProbe:
             exec:
               command:
-              - /usr/bin/pg_isready
+                - bash
+                - "-c"
```
[the remark this answers is not in the document]
Comment (dhirajsb) [reply config/samples/postgres/postgres-db.yaml:76]: Need bash to do the env variable substitution.

**Comment by dhirajsb** [comment]
> Is there a procedure to update or rotate the certificates when they expire?
> Is there a procedure for a user to bring in their custom certificates instead of self-generated?

Good dieas, I'll add them to the readme. In general, these are just samples and users are not expected to use these exact DB configurations in production. The secure DB samples are there to test and demo model registry support for secure DB connections.

**Pull request merged**